portfolio/src/main/webapp/script.js

```diff
@@ -292,11 +293,21 @@ function setupPageByLoginStatus() {
   });
 }
 
+function fetchBlobstoreUrl() {
+  fetch('/blobstore-upload-url').then(response => response.text())
+      .then((imageUploadUrl) => {
+        const inputForm = document.getElementById('input-form');
+        inputForm.action = imageUploadUrl;
+        inputForm.style.display = 'block';
+      });
+}
+
 /** 
- * Checks log in status to set up comments.html when the page is loading; 
- * fetches all the comments and displays them.
+ * Checks log in status to set up comments.html when the page is loading. 
+ * Fetches all the comments and displays them. Fetches the blobstore URL.
  */
 function loadCommentsPage() {
```

Comment:
I think you want to rename this now since you're doing so much more in there.

Reply:
It's hard to think of a good name lol. I call it `initiateCommentsPage` now.